src/applications/claims-status/components/SubmittedTrackedItem.jsx

```diff
@@ -1,31 +1,33 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import moment from 'moment';
+import PropTypes from 'prop-types';
 
 import {
-  truncateDescription,
-  hasBeenReviewed,
   getItemDate,
+  hasBeenReviewed,
+  truncateDescription,
 } from '../utils/helpers';
 
 export default function SubmittedTrackedItem({ item }) {
-  const closed =
-    item.type.startsWith('never_received') ||
-    item.status === 'NO_LONGER_REQUIRED';
+  const { closedDate, description, displayName, documents, status } = item;
+
+  const closed = status === 'NO_LONGER_REQUIRED' && closedDate !== null;
   const reviewed = hasBeenReviewed(item);
 
   return (
     <div className="submitted-file-list-item">
-      <h3 className="submission-file-type">{item.displayName}</h3>
+      <h3 className="submission-file-type">{displayName}</h3>
       <p className="submission-description">
-        {truncateDescription(item.description)}
+        {truncateDescription(description)}
       </p>
-      {item.documents &&
-        item.documents.map((doc, index) => (
+      {documents &&
+        documents.map((doc, index) => (
```

Comment:
should this check length?
```js
documents.length && documents.map(...)
```

Reply:
I think it's just checking that the key exists. If the length is 0 the map just won't generate any jsx